### DISCUSSION TITLE
[Feature] Backup "hidden" albums

### DISCUSSION THREAD

**Discussion by oa01** [immich-app/immich#3447] [Feature Request]
### The feature

I have some of my pictures in hidden albums (on android you can hide folders by putting a . in front of the name).
But it seems like i cannot select these albums in the immich android app.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by HikariNoKitsune** [comment]
would also like this to be a thing. being able to backup everything needed would be great :D

**Comment by daniele-athome** [comment]
I believe Android Media Manager will completely hide those "hidden albums" to other applications - but I might be wrong, I haven't been working on Android for years now.

**Reply by oa01** [reply]
These folders behave just like dotfiles on linux.
Any application can see them

**Reply by daniele-athome** [reply]
I know, I mean that immich probably uses the Android Media Manager which - IIRC - doesn't expose dot folders.
